requirements.txt

```diff
@@ -12,3 +12,4 @@ PyYAML>=3.11
 requests>=2.10.0
 tinysegmenter==0.3  # TODO(codelucas): Investigate making this >=0.3
 tldextract>=2.0.1
+pandas>=0.25.3
```

Comment:
why do this?

Reply:
Mistakenly added, shall remove momentarily